src/main/java/frc/robot/commands/ShooterArmCommands/MoveShooterArmUntil.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.ShooterArmCommands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ShooterArmSubsystem;
+
+public class MoveShooterArmUntil extends Command {
+  /** Creates a new ResetShooterArmPos. */
+  private final ShooterArmSubsystem shooterArmSubsystem = ShooterArmSubsystem.getInstance();
+  
+  public MoveShooterArmUntil() {
+    addRequirements(shooterArmSubsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    shooterArmSubsystem.moveArmByVoltage();
+  }
+
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+    shooterArmSubsystem.setPosition(0);
```

Comment:
set to `resetPose` not to 0

Reply:
did it

Reply:
my bad create a new constant called startPose and set it to that 
also when you create the MM above set it to the start pose

Reply:
did it